examples/wasm-example/src/routes/signing.rs

```diff
@@ -0,0 +1,392 @@
+use anyhow::anyhow;
+use futures::FutureExt;
+
+use subxt::{OnlineClient, PolkadotConfig};
+
+use subxt::ext::codec::{Decode, Encode};
+use subxt::tx::SubmittableExtrinsic;
+use subxt::tx::TxPayload;
+use subxt::utils::{AccountId32, MultiSignature};
+
+use crate::services::{extension_signature_for_partial_extrinsic, get_accounts, polkadot, Account};
+use web_sys::HtmlInputElement;
+use yew::prelude::*;
+
+pub struct SigningExamplesComponent {
+    message: String,
+    remark_call_bytes: Vec<u8>,
+    online_client: Option<OnlineClient<PolkadotConfig>>,
+    stage: SigningStage,
+}
+
+impl SigningExamplesComponent {
+    /// # Panics
+    /// panics if self.online_client is None.
+    fn set_message(&mut self, message: String) {
```

Comment:
Could it be possible that `set_message` is called before the `OnlineClient` is created here?

I'm trying to think of the scenario where the light-client might take a bit to initialize while the user is eager to use the example.

If so, I wonder if we could warn the user somehow to try again later? Maybe a (red) message somewhere that the client is not initialized yet and to try again later?

Reply:
That makes sense, thanks for the info!